fluid-music/src/FluidClient.js

```diff
@@ -1,61 +1,31 @@
-const dgram = require('dgram');
-const osc = require('osc-min');
+const IpcClient = require("osc-ipc-client")
```

Comment:
This requires statement throws an error for me. It looks like it is importing a package not a file. Where is that package? If osc-ipc-client is a file, and not a package, it should begin with `./osc-ipc-client`

```
Error: Cannot find module 'osc-ipc-client'
Require stack:
- /Users/charles/projects/juce/cybr/fluid-music/src/FluidClient.js
- /Users/charles/projects/juce/cybr/fluid-music/src/index.js
- /Users/charles/projects/juce/cybr/demo-stockhausen-cage-xenakis/template.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:954:17)
    at Function.Module._load (internal/modules/cjs/loader.js:847:27)
    at Module.require (internal/modules/cjs/loader.js:1016:19)
    at require (internal/modules/cjs/helpers.js:69:18)
    at Object.<anonymous> (/Users/charles/projects/juce/cybr/fluid-music/src/FluidClient.js:1:19)
    at Module._compile (internal/modules/cjs/loader.js:1121:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1160:10)
    at Module.load (internal/modules/cjs/loader.js:976:32)
    at Function.Module._load (internal/modules/cjs/loader.js:884:14)
    at Module.require (internal/modules/cjs/loader.js:1016:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/charles/projects/juce/cybr/fluid-music/src/FluidClient.js',
    '/Users/charles/projects/juce/cybr/fluid-music/src/index.js',
    '/Users/charles/projects/juce/cybr/demo-stockhausen-cage-xenakis/template.js'
  ]
}
```

Reply:
In the event that you have written code in the node_modules folder - be careful! That content will likely be overwritten whenever you `npm install`, and you don't want to loose your work. `node_modules` is also .gitignore'd so I won't have access to that code. 